src/SchedulerManager/Travely.SchedulerManager.API/appsettings.json

```diff
@@ -21,7 +21,10 @@
       "Protocols": "Http2"
     }
   },
-  "Hangfire": {
-    "ConnectionString": "Server=localhost\\SQLEXPRESS;Database=HangfireTest;Trusted_Connection=True;"
```

Comment:
Incorrect merge please move it to the ConnectionStrings section instead of deleting this part.